src/Tasks/Delegate.cs

```diff
@@ -15,13 +15,13 @@
 namespace Microsoft.Build.Tasks
 {
     /// <summary>
-    /// File.GetAttributes delegate
+    /// File.GetAttributes delegate.
```

Comment:
I think the official guidance is to only include a period if the preceding words form a complete sentence, which isn't true here. That said, I don't really care.

Reply:
Reverted file.